### DISCUSSION TITLE
Keyword of MISC

### DISCUSSION THREAD

**Discussion by lihy213** [firemodels/fds#13948] [General]
Hello team, 
I found that the keyword `SUPPRESSION` was not listed in the `MISC` table but in the source code of MISC namelist. Maybe you could fix it. BTW, what's the meaning of `SUPPRESSION` here in MISC?

**Comment by lihy213** [comment]
Also the same to `STRATIFICATION`.

**Reply by drjfloyd** [reply]
SUPPRESSION was moved from MISC to COMB. In the User's Guide if you search for SUPPRESSION in the tables in the back you will see it on the COMB line which points to Section 13.1.6 in the Guide (it is in footnote 2). Setting SUPPRESSION=F turns off the extinction models.

The preference for new input files is to follow the User's Guide and use the input on the COMB line. We kept it on MISC to provide for a period of backwards compatibility, but at some point that will be removed.

STRATIFICATION is similar. It was moved to the WIND group but kept on MISC for a period of backwards compatibility.

In general we alphabetize the variables in the various namelist definitions in the source. If you see variables at the end of the definition that are out of order then those are likely either experimental parameters not yet released or variables that have been moved or deprecated but kept for a period of backwards compatibility.

**Reply by lihy213** [reply]
Clear! :)
